keras_cv/models/resnet_v1_test.py

```diff
@@ -51,8 +52,54 @@ def test_application_variable_input_channels(self, app, last_dim, args):
         super()._test_application_variable_input_channels(app, last_dim, args)
 
     @parameterized.parameters(*MODEL_LIST)
-    def test_model_can_be_used_as_backbone(self, app, last_dim, args):
-        super()._test_model_can_be_used_as_backbone(app, last_dim, args)
+    def test_model_serialization_tf(self, app, last_dim, args):
+        super()._test_model_serialization(
+            app, last_dim, args, save_format="tf", filename="model"
+        )
+
+    @parameterized.parameters(*MODEL_LIST)
+    def test_model_serialization_keras_format(self, app, last_dim, args):
+        if version.parse(tf.__version__) >= version.parse("2.12.0-dev0"):
+            super()._test_model_serialization(
+                app,
+                last_dim,
+                args,
+                save_format="keras_v3",
+                filename="model.keras",
+            )
+
+    def test_create_backbone_model_from_application_model(self):
```

Comment:
If we do add this type of tests, I think it should be in the base `model_test` and referenced from these tests.

That said, we are not certain exactly how the `as_backbone` API will end up -- it will be at least renamed, and potentially its structure changed slightly. So perhaps for now we should leave out these tests.